tests/test_app/TestApp/Middleware/UnauthorizedHandler/SuppressHandler.php

```diff
@@ -14,13 +15,14 @@
  * @since 1.0.0
  * @license https://opensource.org/licenses/mit-license.php MIT License
  */
+
 namespace TestApp\Middleware\UnauthorizedHandler;
 
 use Authorization\Exception\Exception;
 use Authorization\Middleware\UnauthorizedHandler\HandlerInterface;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
-use Zend\Diactoros\Response;
+use Cake\Http\Response;
```

Comment:
Why did this change?

Reply:
Because PHPUnit tests were failing because of that missing class. The Zend\Diactoros package is not included in the composer.json.

Reply:
@markstory Laminas removed the Zend => Laminas compatibility classes in a recent release.